### DISCUSSION TITLE
Plugins autoinstall

### DISCUSSION THREAD

**Discussion by TheShadowAngel** [doublecmd/doublecmd#1771] [Suggestions]
Доброго времени!
Предложение: было бы не плохо сделать автоустановку плагинов как в TC - если открыть архив с плагином то предлагается установка или обновление.

**Comment by PetbkA** [comment]
👍 

Plugins autodiscovery and installation offer would be great.

Of course:
1. Plugin type should be detected depending on the file extension;
2. Plugin architecture would be selected similar to DC architecture. In particular, if there's no files for current architecture, alert can be shown, e.g. `Only 32-bit plugin is available. Find 64-bit version or use 32-bit DC`;
3. (Optionally or customizable) Plugins list would be opened after installation to customize plugin
